src/components/SideMenu/SiteList/SiteList.tsx

```diff
@@ -5,35 +5,21 @@ interface SiteItemType {
   name: string;
 }
 
-const NavList = async () => {
-  const getAllSites = async (): Promise<SiteItemType[]> => {
-    const response = await fetch(`${process.env.NEXT_PUBLIC_ENDPOINT}/sites`);
+const NavList = async() => {
 
-    if(response.status == 404) return [];
+  const response = await fetch(`${process.env.NEXT_PUBLIC_ENDPOINT}/sites`);
```

Comment:
サーバーサイドでの処理になるため、NEXT_PUBLICがついていない方の環境変数で十分だと思います！
```suggestion
  const response = await fetch(`${process.env.ENDPOINT}/sites`);
```